src/Arcus.Security.Core/Caching/Configuration/CacheConfiguration.cs

```diff
@@ -9,28 +9,31 @@ namespace Arcus.Security.Core.Caching.Configuration
     public class CacheConfiguration : ICacheConfiguration
     {
         /// <summary>
-        ///     Duration for which an entry should be cached
+        /// Initializes a new instance of the <see cref="CacheConfiguration"/> class with default cache entry of 5 minutes.
         /// </summary>
-        public TimeSpan Duration { get; }
+        public CacheConfiguration() : this(TimeSpan.FromMinutes(5))
+        {
+        }
 
         /// <summary>
-        ///     Constructor
+        /// Initializes a new instance of the <see cref="CacheConfiguration"/> class.
         /// </summary>
         /// <param name="duration">Duration for which an entry should be cached</param>
         /// <exception cref="ArgumentOutOfRangeException">Thrown when the cache duration is not a positive time duration.</exception>
         public CacheConfiguration(TimeSpan duration)
         {
             Guard.NotLessThan(duration, TimeSpan.Zero, nameof(duration), "Requires a positive time duration in which the caching should take place");
-
             Duration = duration;
         }
 
         /// <summary>
-        ///     Constructor with default cache entry of 5 minutes
+        /// Gets the default <see cref="ICacheConfiguration"/> that takes in 5 minutes as default cache duration.
         /// </summary>
-        public CacheConfiguration()
-        {
-            Duration = TimeSpan.FromMinutes(5);
-        }
+        public static ICacheConfiguration Default { get; } = new CacheConfiguration();
```

Comment:
Minor remark, and I also don't know if it's worth it, but defining this as a static readonly field, gives you lazy instantiation.  I don't know if this is also the case if you declare it as a property.

```csharp
public static readonly ICacheConfiguration Default = new CacheConfiguration();
```

I think you can test it by putting a breakpoint on the initializer and see if the breakpoint is hit before you first request the `Default` property or not.

Reply:
Yes, normally singletons are field I think? Yeah, we could test that.

Reply:
As I see it, properties are also lazy loaded. We could use fields here as they are generating less IL code but not sure if we should micro-manage something like this in favor of a public property.

Reply:
If static readonly properties are also lazy loaded, I guess we could keep it as it is.